dist/t/spec/features/login_spec.rb

```diff
@@ -1,10 +1,92 @@
+    within("div#subheader") do
+      click_link('Logout')
+    end
   end
-
 end
+
+RSpec.describe "Create Interconnect as admin and build pckg" do
+  it "should be able to add Opensuse Interconnect as Admin" do
+    visit "/user/login"
+    fill_in 'user_login', with: 'Admin'
+    fill_in 'user_password', with: 'opensuse'
+    save_screenshot("/tmp/login/l.png")
+    click_button('Log In »')
+    visit "/configuration/interconnect"
+    click_button('openSUSE')
+    click_button('Save changes')
+  end
+
+  it "should be able to create home project" do
+    click_link('Create Home')
+    expect(page).to have_content("Create New Project")
+    find('input[name="commit"]').click #Create Project
+    expect(page).to have_content("Project 'home:Admin' was created successfully")
+  end
+
+  it "should be able to create a new package from OBS:Server:Unstable/build/build.spec" do
+    File.write("build.spec", Net::HTTP.get(URI.parse("https://api.opensuse.org/public/source/OBS:Server:Unstable/build/build.spec")))
+    find('img[title="Create package"]').click
+    expect(page).to have_content("Create New Package for home:Admin")
+    fill_in 'name', with: 'obs-build'
+    find('input[name="commit"]').click #Save changes
+    expect(page).to have_content("Package 'obs-build' was created successfully")
+    find('img[title="Add file"]').click
+    expect(page).to have_content("Add File to")
+    attach_file("file", "build.spec")
+    find('input[name="commit"]').click #Save changes
+    expect(page).to have_content("Source Files")
+  end
+
+  it "should be able to attach _service file" do
+    File.write("_service", Net::HTTP.get(URI.parse("https://api.opensuse.org/public/source/OBS:Server:Unstable/build/_service")))
+    find('img[title="Add file"]').click
+    expect(page).to have_content("Add File to")
+    attach_file("file", "_service")
+    find('input[name="commit"]').click #Save changes
+    expect(page).to have_content("Source Files")
+  end
+
+  it "should be able to add build targets from existing repos" do
+    click_link('build targets')
+    expect(page).to have_content("openSUSE distributions")
+    check('repo_openSUSE_Tumbleweed')
+    check('repo_openSUSE_Leap_42.1')
+    find('input[id="submitrepos"]').click #Add selected repositories
+    expect(page).to have_content("Successfully added repositories")
+    expect(page).to have_content("openSUSE_Leap_42.1 (x86_64)")
+    expect(page).to have_content("openSUSE_Tumbleweed (i586, x86_64)")
+  end
+
+  it "should be able to Overview Build Results" do
+    click_link('Overview')
+    expect(page).to have_content("Build Results")
+  end
+
+  it "should be able to refresh Build Results" do
+    #TO DO research osc buildlog option to gather results
+    timeout = 60
+    build_succeeded = false
+    until timeout == 0 do
+      click_link('Build Results')
+      if expect(page).to have_content("succeeded")
+        build_succeeded = true
+        break
+      timeout -= 1
+    end
```

Comment:
I think you should work with time here. At some point this will always be longer than you expect it. A more intelligent way to `wait for the build to succeed` would be to work with the live build log...